docs/user/integration/integrate-jaeger.md

```diff
@@ -0,0 +1,181 @@
+# Install custom Jaeger in Kyma
+
+## Overview
+
+The following instructions outline how to use [`Jaeger`](https://github.com/jaegertracing/helm-charts/tree/main/charts/jaeger) as a tracing backend with Kyma's [TracePipeline](https://kyma-project.io/#/telemetry-manager/user/03-traces).
+
```

Comment:
The dynatrace-sample-app folder should be renamed to "integrate-dynatrace" to stay consistent

Reply:
Excellent! I wasn't sure if I may deviate from the original folder name too much.

Reply:
all folders are named consistently now